.eslintrc.js

```diff
@@ -2,16 +2,56 @@
 // if you're looking for a client (especially React) or server lint rule
 // go to the configs in their respective directories
 module.exports = {
+	env: { 'jest/globals': true },
```

Comment:
I think a better solution rather than enabling jest globals everywhere is to take advantage of eslint being js and enable it only for files that match the *.test.tsx regex: https://stackoverflow.com/a/49211283/5041889